src/index.ts

```diff
@@ -231,6 +231,10 @@ export class Snackbar {
   }
 
   expand() {
+    // Do not expand when hovering on hidden snackbar
+    if (this.el !== undefined && this.el.style.opacity === '0') {
+      return
+    }
```

Comment:
hmm this did fix the issue but the hidden snackbar will still overlap other elements